docs/doc/11-integrations/30-file-format-options.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Input File Formats
```

Comment:
most options works for both input/format.
Some only for input/output should be pointed output alone.

Reply:
Could you tell me which ones are for input or output ONLY?

Reply:
https://github.com/datafuselabs/databend/blob/main/docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md#skip_header--integer

XML format is now input only

CSV quote and skip_header options is  input only

Reply:
snowflake docs use  **Usage**  for this

Reply:
The "output" refers to unloading data from Databend, right? @youngsofun

Reply:
> The "output" refers to unloading data from Databend, right? @youngsofun 

yes